<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamRoomDialog</class>
 <widget class="QWidget" name="ExamRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Exam Room Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabExamRoomDetail">
      <attribute name="title">
       <string>Chi tiet phong thi</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Tên phòng thi</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Mô tả</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Time limit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ngày bắt đầu</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ngày kết thúc</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Trạng thái</string>
       </property>
      </widget>
      <widget class="QLabel" name="roomName">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Thi cuoi ky lap trinh mang</string>
       </property>
      </widget>
      <widget class="QLabel" name="description">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>701</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Mo ta</string>
       </property>
      </widget>
      <widget class="QLabel" name="timelimit">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>60 phút</string>
       </property>
      </widget>
      <widget class="QLabel" name="starttime">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>100</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>2024-11-12 10:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="endtime">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>130</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>2024-11-14 10:00:00</string>
       </property>
      </widget>
      <widget class="QLabel" name="status">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Chưa bắt đầu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="startExam">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>180</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Phát đề</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ngân hàng câu hỏi</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addQuestionButton">
       <property name="geometry">
        <rect>
         <x>780</x>
         <y>220</y>
         <width>111</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Thêm câu hỏi</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listQuestionBankWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>891</width>
         <height>271</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabUser">
      <attribute name="title">
       <string>Danh sach nguoi tham gia</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Danh sách người trong phòng thi</string>
       </property>
      </widget>
      <widget class="QWidget" name="searchBar" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>782</width>
         <height>43</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="searchLabel">
          <property name="text">
           <string>Search User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="searchInput"/>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="addUser">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>10</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Thêm User</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="userTableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>782</width>
         <height>684</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>userId</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>email</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>dob</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>view</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tabExamResult">
      <attribute name="title">
       <string>Ket qua cua phong thi</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>241</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Danh sách kết quả phòng thi</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="examResultTableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>782</width>
         <height>684</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>userId</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>email</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>so cau dung</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>tong so cau</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>diem</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tabExamResultChart">
      <attribute name="title">
       <string>Thong ke pho diem</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
